app/javascript/articles/Article.jsx

```diff
@@ -0,0 +1,136 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import { articlePropTypes } from '../src/components/common-prop-types/article-prop-types';
+import {
+  CommentsCount,
+  ContentTitle,
+  OrganizationHeadline,
+  PublishDate,
+  ReadingTime,
+  SaveButton,
+  SearchSnippet,
+  TagList,
+  ReactionsCount,
+} from './components';
+import { PodcastArticle } from './PodcastArticle';
+
+/* global timeAgo */
+
+export const Article = ({
+  article,
+  currentTag,
+  isBookmarked,
+  reactionsIcon,
+  commentsIcon,
+  videoIcon,
+}) => {
+  if (article && article.type_of === 'podcast_episodes') {
+    return <PodcastArticle article={article} />;
+  }
+
+  return (
+    <div
+      className="single-article single-article-small-pic"
+      data-content-user-id={article.user_id}
+    >
+      {article.cloudinary_video_url && (
+        <a
+          href={article.path}
+          className="single-article-video-preview"
+          style={`background-image:url(${article.cloudinary_video_url})`}
+        >
+          <div className="single-article-video-duration">
+            <img src={videoIcon} alt="video camera" loading="lazy" />
+            {article.video_duration_in_minutes}
+          </div>
+        </a>
+      )}
+
+      <OrganizationHeadline organization={article.organization} />
+      <div className="small-pic">
+        <a
+          href={`/${article.user.username}`}
```

Comment:
As mentioned in Slack @joshpuetz @benhalpern, the feed is returning article entities, but they no longer seem to have `user` or `organization` properties. The same information is accessible via `article.cached_user.table` or `article.cached_organization.table`.

Reply:
I'm not sure...the original feed erb fragment must have converted `article.cached_user` to `.user`